client.js

```diff
@@ -1,41 +1,141 @@
-function initiatePayment() {
-    // prevent multiple clicking
-    unbindClick();
+'use strict';
 
+var elementId = "tokenPayBtn";
+var tokenController;
+var button;
+
+function clean() {
+    if (button) {
+        button.destroy();
+        button = null;
+    }
+
+    if (tokenController && tokenController.destroy) {
+        tokenController.destroy();
+        tokenController = null;
+    }
+}
+
+function createRedirectButton() {
+    // clean up instances
+    clean();
+
+    // create TokenPopupController to handle Popup messages
+    tokenController = window.Token.createRedirectController();
+
+    // get button placeholder element
+    var element = document.getElementById(elementId);
+
+    // create the button
+    button = window.Token.createTokenButton(element, {
+        label: "Redirect Token Quick Checkout",
+    });
+
+    // bind the Token Button to the Redirect Controller when ready
+    tokenController.bindButtonClick(button, function(action) {
```

Comment:
do we need onload here too?

Reply:
I'm not entirely sure why, but when I add onLoad to this one, it doesn't work...